ras_party/controllers/queries.py

```diff
@@ -649,21 +650,24 @@ def query_respondent_enrolments(
     session: session, respondent_id: int, business_id: UUID = None, survey_id: UUID = None, status: int = None
 ) -> list[Enrolment]:
     """
-    Query to return a list of respondent Enrolments. Business_id, survey_id and status can also be added as conditions
+    Query to return a list of respondent Enrolments and business attributes.
+    Business_id, survey_id and status can also be added as conditions
     """
-    additional_conditions = []
+    where_clause = f"WHERE respondent_id = {respondent_id}"
 
     if business_id:
-        additional_conditions.append(Enrolment.business_id == business_id)
+        where_clause += f" and partysvc.enrolment.business_id='{business_id}'"
     if survey_id:
-        additional_conditions.append(Enrolment.survey_id == survey_id)
+        where_clause += f" and partysvc.enrolment.survey_id='{survey_id}'"
     if status:
-        additional_conditions.append(Enrolment.status == status)
-
-    return (
-        session.query(Enrolment, Business.business_ref, BusinessAttributes.attributes)
-        .join(Business, Business.party_uuid == Enrolment.business_id)
-        .join(BusinessAttributes, BusinessAttributes.business_id == Enrolment.business_id)
-        .filter(and_(Enrolment.respondent_id == respondent_id, *additional_conditions))
-        .all()
+        where_clause += f" and partysvc.enrolment.status='{status}'"
+
+    return session.execute(
+        text(
+            f"SELECT * FROM partysvc.enrolment "
```

Comment:
This would benefit from explicitly returning the table fields we want to map to the Python object rather than wilcarding all the fields.